### PR TITLE
i18n(zh-cn): translate unknown-csserror.mdx,csssyntax-error.mdx

### PR DESCRIPTION
**Pull request by fuguid**
#### What kind of changes does this PR include?

- New or updated content
- Translated content

#### Description

- What does this PR change? Give us a brief description.
Translate `unknown-csserror.mdx` and `csssyntax-error.mdx `content to Chinese

### CODE REVIEW

File: src/content/docs/zh-cn/reference/errors/unknown-csserror.mdx [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+import DontEditWarning from '~/components/DontEditWarning.astro'
+
+<DontEditWarning />
+
+
+## 哪里出了问题？
+Astro 在解析您的 CSS 时遇到了未知错误。这通常是由语法错误引起的，错误信息应该包含更多信息。
+
+**参考:**
+-  [样式和 CSS](/en/guides/styling/)
```
Comment (liruifengv) [inline src/content/docs/zh-cn/reference/errors/unknown-csserror.mdx:15]:
```suggestion
-  [样式和 CSS](/zh-cn/guides/styling/)
```

File: src/content/docs/zh-cn/reference/errors/csssyntax-error.mdx [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+
+<DontEditWarning />
+
+
+> **错误信息示例:**<br/>
+CSSSyntaxError: 缺少分号<br/>
+CSSSyntaxError: 未闭合的字符串<br/>
+
+## 哪里出了问题？
+由于语法错误，Astro 在解析您的 CSS 时遇到了错误。通常是由于缺失分号引起的。
```
Comment (liruifengv) [inline src/content/docs/zh-cn/reference/errors/csssyntax-error.mdx:16]:
```suggestion
由于语法错误，Astro 在解析你的 CSS 时遇到了错误。通常是由于缺失分号引起的。
```

File: src/content/docs/zh-cn/reference/errors/unknown-csserror.mdx [hunk 3]
```diff
@@ -0,0 +1,15 @@
...
+i18nReady: true
+githubURL: https://github.com/withastro/astro/blob/main/packages/astro/src/core/errors/errors-data.ts
+---
+import DontEditWarning from '~/components/DontEditWarning.astro'
+
+<DontEditWarning />
+
+
+## 哪里出了问题？
+Astro 在解析您的 CSS 时遇到了未知错误。这通常是由语法错误引起的，错误信息应该包含更多信息。
```
Comment (liruifengv) [inline src/content/docs/zh-cn/reference/errors/unknown-csserror.mdx:12]:
```suggestion
Astro 在解析你的 CSS 时遇到了未知错误。这通常是由语法错误引起的，错误信息应该包含更多信息。
```

File: src/content/docs/zh-cn/reference/errors/csssyntax-error.mdx [hunk 4]
```diff
@@ -0,0 +1,16 @@
...
+i18nReady: true
+githubURL: https://github.com/withastro/astro/blob/main/packages/astro/src/core/errors/errors-data.ts
+---
+import DontEditWarning from '~/components/DontEditWarning.astro'
+
+<DontEditWarning />
+
+
+> **错误信息示例:**<br/>
+CSSSyntaxError: 缺少分号<br/>
```
Comment (liruifengv) [inline src/content/docs/zh-cn/reference/errors/csssyntax-error.mdx:12]: I don't think there's any need for a translation here.

File: src/content/docs/zh-cn/reference/errors/csssyntax-error.mdx [hunk 5]
```diff
@@ -0,0 +1,16 @@
...
+CSSSyntaxError: 未闭合的字符串<br/>
```
Comment (liruifengv) [inline src/content/docs/zh-cn/reference/errors/csssyntax-error.mdx:13]: I don't think there's any need for a translation here.

**Review by liruifengv — CHANGES_REQUESTED** [review]

File: src/content/docs/zh-cn/reference/errors/csssyntax-error.mdx (hunk 5 above)
Comment (fuguid) [reply src/content/docs/zh-cn/reference/errors/csssyntax-error.mdx:13]: Okay, I'll make some modifications.

File: src/content/docs/zh-cn/reference/errors/csssyntax-error.mdx [hunk 6]
```diff
@@ -0,0 +1,16 @@
...
+title: CSS Syntax Error
+i18nReady: true
+githubURL: https://github.com/withastro/astro/blob/main/packages/astro/src/core/errors/errors-data.ts
+---
+import DontEditWarning from '~/components/DontEditWarning.astro'
+
+<DontEditWarning />
+
+
+> **Example error messages:**<br/>
```
Comment (liruifengv) [inline src/content/docs/zh-cn/reference/errors/csssyntax-error.mdx:11]:
```suggestion
> **错误信息示例：**<br/>
```
Translation needs here.

**Review by liruifengv — APPROVED** [review]
LGTM!

File: src/content/docs/zh-cn/reference/errors/unknown-csserror.mdx [hunk 7]
```diff
@@ -0,0 +1,15 @@
+---
+title: Unknown CSS Error
+i18nReady: true
+githubURL: https://github.com/withastro/astro/blob/main/packages/astro/src/core/errors/errors-data.ts
+---
+import DontEditWarning from '~/components/DontEditWarning.astro'
+
+<DontEditWarning />
```
Comment (Yan-Thomas) [inline src/content/docs/zh-cn/reference/errors/unknown-csserror.mdx:8]:
```suggestion
```

File: src/content/docs/zh-cn/reference/errors/csssyntax-error.mdx [hunk 8]
```diff
@@ -0,0 +1,16 @@
+---
+title: CSS Syntax Error
+i18nReady: true
+githubURL: https://github.com/withastro/astro/blob/main/packages/astro/src/core/errors/errors-data.ts
+---
+import DontEditWarning from '~/components/DontEditWarning.astro'
+
+<DontEditWarning />
```
Comment (Yan-Thomas) [inline src/content/docs/zh-cn/reference/errors/csssyntax-error.mdx:8]:
```
```

**Review by Yan-Thomas — APPROVED** [review]
LGTM! Thanks for this @fuguid 🙌

**Pull request merged**